Formula/lua.rb

```diff
@@ -80,68 +79,68 @@ def caveats
   end
 
   test do
-    system "#{bin}/lua", "-e", "print ('Ducks are cool')"
+    assert_match "Homebrew is awesome!", shell_output("#{bin}/lua -e \"print ('Homebrew is awesome!')\"")
   end
 end
 
 __END__
 diff --git a/Makefile b/Makefile
-index 7fa91c8..a825198 100644
+index 416f444..1424dba 100644
 --- a/Makefile
 +++ b/Makefile
-@@ -41,7 +41,7 @@ PLATS= aix bsd c89 freebsd generic linux macosx mingw posix solaris
+@@ -41,7 +41,7 @@ PLATS= guess aix bsd c89 freebsd generic linux linux-readline macosx mingw posix
```

Comment:
Looks like `solaris` is still missing here - does it need to be added back?

Reply:
This won't impact the git patch. (more just like reference line)